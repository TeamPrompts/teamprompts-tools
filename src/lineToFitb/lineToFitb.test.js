const lineToFitb = require('./lineToFitb');

test('lineToFitb', () => {
  const line =
    'I am writing something here about ["topic","golf"] and the key question is ["question","whether we will go to the party"].';

  const fitb = lineToFitb({ id: Date.now().toString(), line });

  expect(fitb).toMatchSnapshot({ id: expect.any(String) });
});
