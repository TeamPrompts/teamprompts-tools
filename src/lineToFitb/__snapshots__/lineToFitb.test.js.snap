// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lineToFitb 1`] = `
Object {
  "examples": Array [
    "golf",
    "whether we will go to the party",
  ],
  "id": Any<String>,
  "prompts": Array [
    "topic",
    "question",
  ],
  "source": "I am writing something here about __BLANK__ and the key question is __BLANK__.",
  "tags": Array [],
}
`;
